daml-model/daml/Model/LoanConfig.daml

```diff
@@ -0,0 +1,105 @@
+--
+-- Copyright (c) 2022, Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+--
+
+module Model.LoanConfig where
+
+import Model.Balance
+import DA.Optional
+
+type Amount = Decimal
+type IBAN = Text
+
+{-
+Loan configuration on account (e.g. Provider/IBAN) level
+The idea is to provide the {accountHolder} with individual loans 
+up until {balanceLoanThreshold} is reached and as long as {maxLoanThreshold} 
+has not been reached (as tracked by {activeLoan}).
+-}
+template LoanConfig
+    with
+        provider: Party
+        -- ^ Account provider
+        accountHolder: Party
+        -- ^ Account holder
+        account: IBAN
+        -- ^ Account number
+        balanceLoanThreshold: Amount
+        -- ^ Defines the balance up until which loans will be provided
+        maxLoanThreshold: Amount
```

Comment:
I'd suggest naming this differently, as this is not really a threshold.

something like maxLoan or similar.

Reply:
Done